apps/api/src/modules/auth/misc/jwt.service.ts

```diff
@@ -0,0 +1,82 @@
+import { JwtDto } from '@modela/dtos'
+import { ForbiddenException, Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import { PassportStrategy } from '@nestjs/passport'
+import { ExtractJwt, Strategy } from 'passport-jwt'
+import { PrismaService } from 'src/database/prisma.service'
+
+import { AuthService } from '../auth.service'
+
+const cookieExtractor = function (req) {
+  if (req && req.cookies) {
+    return req.cookies['token']
+  }
+  return null
+}
```

Comment:
(Optional)
I think it might better if we refactor to utils module just in case that we need to reuse this